CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.1.0] - 2019-01-15
+
+### Added
+- Update all apps to use data sets & releases
+- Improve Docker support
+- Added HURUmap-land app
+- Added Afrobarometer Round 7 survey for Uganda and Zambia
+- Added CHANGELOG
+
+### Changed
+- Hide Empty sections on the profile page
```

Comment:
I've tested for Kenya only but it still shows empty sections.

![screenshot from 2019-01-16 10-36-05](https://user-images.githubusercontent.com/1779590/51233323-b0893f80-197a-11e9-8256-fcf4b0ea44a9.png)